docs/guide/advanced/slots.md

```diff
@@ -134,8 +134,6 @@ test('layout full page layout', () => {
 })
 ```
 
-Note: passing a component using `{ template: '<div /> }` is not supported. Use a HTML string, render function, plain text, or an SFC.
```

Comment:
What do you think about adding an example? Just adding a simple case above (lines 127 and 133) would do

Reply:
Does anyone remember why this was not supported? 🤔

Reply:
@afontcu makes sense, will do it ASAP! Also found a couple more places to clean up comments :)

Reply:
LMK when this is done, we can merge it up. Thanks!

Reply:
@lmiller1990 @afontcu Done :rocket: !